src/services/NotificationService.ts

```diff
@@ -0,0 +1,30 @@
+import { NotificationData } from "types/notifications"
+
+import { apiService } from "./ApiService"
+
+export const getNotifications = async ({
+  siteName,
+}: {
+  siteName: string
+}): Promise<NotificationData[]> => {
+  const endpoint = `/sites/${siteName}/notifications`
+  return apiService.get(endpoint).then((res) => res.data)
+}
+
+export const getAllNotifications = async ({
+  siteName,
+}: {
+  siteName: string
+}): Promise<NotificationData[]> => {
+  const endpoint = `/sites/${siteName}/notifications/allNotifications`
```

Comment:
Should we use query params instead and use the same notifications route as above?

Reply:
I'm okay with this option, but I'm wondering if it's worth doing now - do you feel like this helps the readability of the code?

Reply:
Yup I don't think it's worth doing it now since it's just a minor design preference.